Umweltzone/src/main/res/values-da/strings.xml

```diff
@@ -0,0 +1,320 @@
+        Grundet at mange offentlige udgivelser af scannede landkort er i meget lav opl&#248;sning kan det ikke garanteres, at zonernes kanter er placeret pr&#230;cist s&#229;dan som de er i virkeligheden. Zonernes data og koordinater kan variere fra kilde til kilde omkring kvaliteten og pr&#230;cisionen i digitaliserings processen der er blevet anvendt. Derudover er der ingen mekanisme/procesdure for, at blive opm&#230;rksom p&#229; opdateringer lavet af byen/staten til zonernes data og koordinater. Zone data bliver opdateret manuelt og afh&#230;nger kratigt af venlige brugeres tilbagemeldinger (som dig f.eks.).
+    </string>
+    <string name="appinfo_references_name_umweltbundesamt">Federal Environment Agency</string>
+
+    <!-- AboutActivity / Graphics -->
+    <string name="appinfo_references_graphics_title">Grafik</string>
+    <string name="appinfo_references_graphics_content">
+        Grafik s&#229;som logoet og klisterm&#230;rker er taget fra Wikimedia Commons.
+    </string>
+
+    <!-- AboutActivity / Texts -->
+    <string name="appinfo_references_texts_title">Tekster</string>
+    <string name="appinfo_references_texts_content">
+        En kilde reference bliver givet n&#229;r en tekst bliver taget fra et andet sted.
+    </string>
+
+    <!-- AboutActivity / Libraries -->
+    <string name="appinfo_references_libraries_title">Biblioteker</string>
+    <string name="appinfo_references_libraries_content">
+        F&#248;lgende software biblioteker bliver brugt i denne applikation.
+    </string>
+
+    <!-- AboutActivity / Sourcecode -->
+    <string name="appinfo_sourcecode_title">Kilde kode</string>
+    <string name="appinfo_sourcecode_content">
+        Kilde koden til applikationen kan findes p&#229; nettet.
+    </string>
+    <string name="appinfo_sourcecode_url_title">Applikations kilde kode</string>
+
+    <!-- AboutActivity / License -->
+    <string name="appinfo_license_title">Licenser</string>
+    <string name="appinfo_license_content">
+        Kilde koden for Umweltzone Appen bruger GPLv3 Licensen.
+        Geometri koordinaterne er taget fra OpenStreetMap som bruger Open Database Licencen (ODbL) v1.0. Andet information fra andre kilder bruger licensen beskrevet hos dem. Billede og tekst bruger The Creative Commons Attribution 3.0 Unported Licens.
+    </string>
+    <string name="appinfo_license_url_title_gpl">GPLv3 Licens</string>
+    <string name="appinfo_license_url_title_creative_commons">Creative Commons Licens</string>
+
+    <!-- AboutActivity / Build Info -->
+    <string name="appinfo_build_info_title">Build information</string>
+    <string name="appinfo_build_info_time">Build time</string>
+    <string name="appinfo_build_info_version_code">Version Kode</string>
+    <string name="appinfo_build_info_hash">Build Hash</string>
+
+
+    <!-- FeedbackActivity / Contact -->
+    <string name="appinfo_contact_title">Kontakt</string>
```

Comment:
Is this Danish?

Reply:
It is Danish, but on second thought I added the "Kontakt os" which means "contact us" because you don't simply say "kontakt". Hope it's okay